python-modules/chevah-python-test/test_python_binary_dist.py

```diff
@@ -421,7 +421,7 @@ def main():
 
             if chevah_os == 'windows':
                 # Check OpenSSL version on windows.
-                expecting = u'OpenSSL 1.0.2g  1 Mar 2016'
+                expecting = u'OpenSSL 1.1.0f  25 May 2017'
```

Comment:
Are you sure having different major OpenSSL versions in `stdlib` and `cryptography` is not an issue? I looked it up a bit, and the `cryptography` guys thinks it's fine for Windows and MacOS (at https://cryptography.io/en/latest/installation/#static-wheels).

Anyway, this is how it looks now:
```
##### Executing Chevah tests... #####
python 2.7.13 (v2.7.13:a06454b1afa1, Dec 17 2016, 20:42:59) [MSC v.1500 32 bit (Intel)]
zlib 1.0
stdlib ssl OpenSSL 1.0.2j 26 Sep 2016
cryptography 1.9 - OpenSSL OpenSSL 1.1.0f  25 May 2017
pyopenssl 17.0.0 - OpenSSL OpenSSL 1.1.0f  25 May 2017
PyCrypto 2.6.1
ctypes 1.1.0
```